src/js/api/order.js

```diff
@@ -2052,6 +2052,56 @@ ripe.Ripe.prototype.setOrderStatus = function(number, status, options, callback)
     return this._cacheURL(options.url, options, callback);
 };
 
+/**
+ * Sets the Order size in its details.
+ *
+ * @param {Number} number The number of the order to set the size.
+ * @param {string} size The size.
+ * @returns {XMLHttpRequest} The XMLHttpRequest instance of the API request.
+ */
+ripe.Ripe.prototype.setOrderSize = function(number, size, options, callback) {

```

Comment:
Rename it to `setSizeOrder` as we do with all methods. The `setOrderStatus` is an outlier that sadly we must maintain for retrocompatibility.